src/test/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/activities/ActivitiesMigrationIntTest.kt

```diff
@@ -395,19 +395,6 @@ class ActivitiesMigrationIntTest : SqsIntegrationTestBase() {
           migrationType = ACTIVITIES,
         ),
       )
-      migrationHistoryRepository.save(
-        MigrationHistory(
-          migrationId = migrationIdNoStarDate,
-          whenStarted = LocalDateTime.parse("2024-12-05T09:58:45"),
-          whenEnded = LocalDateTime.parse("2024-11-05T10:04:45"),
-          status = MigrationStatus.COMPLETED,
-          estimatedRecordCount = 8,
-          filter = """{"prisonId":"BLI"}""",
-          recordsMigrated = 8,
-          recordsFailed = 0,
-          migrationType = ACTIVITIES,
-        ),
-      )
```

Comment:
No need to test ending migrations without a start date, this doesn't happen anymore.